chain-impl-mockchain/benches/tally.rs

```diff
@@ -0,0 +1,168 @@
+use chain_impl_mockchain::testing::scenario::template::WalletTemplateBuilder;
+use chain_impl_mockchain::{
+    certificate::{EncryptedVoteTally, VotePlan, VoteTally},
+    fee::LinearFee,
+    header::BlockDate,
+    testing::{
+        build_tally_decrypt_share,
+        data::CommitteeMembersManager,
+        ledger::ConfigBuilder,
+        scenario::{prepare_scenario, proposal, vote_plan, wallet},
+        VoteTestGen,
+    },
+    value::Value,
+    vote::{Choice, PayloadType},
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use rand_chacha::ChaCha20Rng;
+use rand_core::SeedableRng;
+
+const ALICE: &str = "Alice";
+const STAKE_POOL: &str = "stake_pool";
+const VOTE_PLAN: &str = "fund1";
+const VOTERS_COUNT: usize = 250;
+
+fn voters_aliases(count: usize) -> Vec<String> {
+    let mut counter = 0;
+    std::iter::from_fn(|| {
+        counter += 1;
+        Some(format!("voter_{}", counter))
+    })
+    .take(count)
+    .collect()
+}
+
+fn tally_benchmark(c: &mut Criterion) {
+    const MEMBERS_NO: usize = 3;
+    const THRESHOLD: usize = 2;
+    let favorable = Choice::new(1);
+
+    let mut wallets: Vec<&mut WalletTemplateBuilder> = Vec::new();
+
+    let mut alice_wallet_builder = wallet(ALICE);
+    alice_wallet_builder
+        .with(1_000)
+        .owns(STAKE_POOL)
+        .committee_member();
+    wallets.push(&mut alice_wallet_builder);
+
+    let voters_aliases = voters_aliases(VOTERS_COUNT);
+    let mut wallet_builders: Vec<WalletTemplateBuilder> =
+        voters_aliases.iter().map(|alias| wallet(alias)).collect();
+
+    for wallet_builder in wallet_builders.iter_mut() {
+        wallet_builder.with(10);
```

Comment:
IMO we should increase stakes to be more realistic. I understand that this is going to slow down the tallying process as it is now, but this should not be the case after offline tallying is merged.